src/main/java/com/problem1/myproject/service/implementation/UserService.java

```diff
@@ -24,12 +23,14 @@
 public class UserService implements IUserService, UserDetailsService {
 
     private UserRepository userRepo;
-    @Autowired
+
+    //@Autowired
     private PasswordEncoder passwordEncoder;
 
     @Autowired
-    public UserService(UserRepository userRepo) {
+    public UserService(UserRepository userRepo, PasswordEncoder passwordEncoder ) {
```

Comment:
@RobertRodeapps Please replace this constructor with a AllArgsConstructor annotation